core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.ts

```diff
@@ -313,6 +313,8 @@ export class TopNavigationBarComponent implements OnInit, OnDestroy {
     this.directiveSubscriptions.add(
       this.windowDimensionsService.getResizeEvent().subscribe(evt => {
         this.windowIsNarrow = this.windowDimensionsService.isWindowNarrow();
+        this.windowRef.nativeWindow.document.body.style.overflowY =
+          this.windowIsNarrow ? 'auto' : '';
```

Comment:
Can this be achieved in the HTML?

Reply:
I have used CSS, but it does not seem to be responsive to changes in screen size

Reply:
We seem to be closing the sidebar below in line 333 regardless of screen size. In that case, should we just set this property to "auto" irrespective of "windowIsNarrow" (we use "auto" when the sidebar is hidden in toggleSidebar function)?

Reply:
Also, would be nice to move this below line 333 so that all related sidebar operations are together.

Reply:
Done